rustls-postquantum/src/lib.rs

```diff
@@ -0,0 +1,231 @@
+//! This crate provides a [`rustls::crypto::CryptoProvider`] that includes
+//! a hybrid[^1], post-quantum-secure[^2] key exchange algorithm --
+//! specifically [X25519Kyber768Draft00].
+//!
+//! X25519Kyber768Draft00 is pre-standardization, so you should treat
+//! this as experimental.  You may see unexpected interop failures, and
+//! the algorithm implemented here may not be the one eventually
+//! becomes widely deployed.
```

Comment:
typo: missing a word here I think:

```suggestion
//! the algorithm implemented here may not be the one that eventually
//! becomes widely deployed.
```